src/web/mjs/connectors/AniGliScans.mjs

```diff
@@ -0,0 +1,26 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+
+export default class AniGliScans extends WordPressMangastream {
+
+    constructor() {
+        super();
+        super.id = 'anigliscans';
+        super.label = 'AniGliScans';
+        this.tags = [ 'manga', 'english' ];
+        this.url = 'https://anigliscans.com';
+        this.path = '/series/?list';
+    }
+    async _getPages(chapter) {
+        const data = await super._getPages(chapter);
+        return data.map(element => this.createConnectorURI(element));
+    }
+    async _handleConnectorURI(payload) {
+        let request = new Request(payload, this.requestOptions);
```

Comment:
Its good. You could change let for const.

Reply:
Done, will update the changes